brave/brave-core
# fix: stream transaction parsed values

Resolves https://github.com/brave/brave-browser/issues/29272

- Fixes a bug where the activity screen only loads transactions for the last-selected network for each coin-type
- Speeds up Transaction rendering by allowing the UI to partially render before parsing dependencies are available



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Create a wallet
2. submit a transaction on an EVM chain
3. switch to a different EVM chain
4. open/refresh the Activity (transactions) page
5. the transactions for both EVM networks should load

Repeat twice, replacing EVM networks with Filecoin and Solana networks
